cart/src/main/java/com/yas/cart/exception/ApiExceptionHandler.java

```diff
@@ -47,6 +47,16 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(MethodArgumentNotV
     return ResponseEntity.badRequest().body(errorVm);
   }
 
+  @ExceptionHandler(Exception.class)
+  protected ResponseEntity<Object> handleOtherException(Exception ex, WebRequest request) {
```

Comment:
Can we use `ResponseEntity<ErrorVm>` like other exception handlers?

Reply:
I changed all the ResponseEntity<Object> to <ErrorVm> in @ExceptionHandle to consistency